docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,174 @@
+---
+meta:
+  title: "DevTools"
+  parentTitle: "Guides"
+  description:
+    "Learn how to leverage Liveblocks DevTools to inspect and troubleshoot your
+    collaborative application."
+---
+
+Our [DevTools](/devtools) are a browser-based extension that integrates with
+Liveblocks and your local development environment. This allows you to easily
+inspect, visualize, and troubleshoot your collaborative online experiences.
+
+DevTools are currently only available if you are on Liveblocks 1.0 or later. To
+view step-by-step instructions on how to upgrade your project to Liveblocks 1.0,
+check out our [upgrade guides](/guides/upgrading/).
+
+<Banner title="Live demo video">
+
+If you want to see how this works in video, be sure to check out this
+[live demo](https://www.youtube.com/watch?v=-6Jxejp0iuw) on YouTube.
+
+</Banner>
+
+## How to use DevTools
+
+### Install the browser extension
+
+Our DevTools extension is available for all major web browsers:
+
+- [Chrome](https://chrome.google.com/webstore/detail/liveblocks-devtools/iiagocfmmhknpdalddkbiejnfmbmlffk)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/liveblocks-devtools/)
+- [Edge](https://microsoftedge.microsoft.com/addons/detail/liveblocks-devtools/hfecmmnilleegmjaegkjjklnjbgadikg)
+
+### Features and tips
+
+Within the "Liveblocks" panel, you will see two panes: storage and presence. The
+storage pane represents each object that you have defined in storage as a tree,
+likewise, the presence pane represents each user that is connected to your
+Liveblocks project.
```

Comment:
I feel this part should be mopved to the "[Install the browser extension](/docs/guides/devtools#Install-the-browser-extension)" section.

This install section feels kind of empty.

And why don't we split it like that?
1. **Install the Liveblocks DevTools**, with what we have in "Install the browser extension" + the firdst paragraph "Feature and tips"
2. **Try the DevTools now**
   - With our Starter Kit
   - With an example from our gallery
3. **How to use DevTools**, with the "Features and tips" sections
4. **Tips ands tricks**

The flow should be: how we install it, how can I test it, how to best use it.

I know you and @nvie already discussed the structure so let me know if that doesn't make sense